k9mail/src/main/java/com/fsck/k9/activity/MessageCompose.java

```diff
@@ -1205,12 +1219,12 @@ private void processSourceMessage(MessageViewInfo messageViewInfo) {
                     break;
                 }
             }
-        } catch (MessagingException me) {
+        } catch (MessagingException | IOException e) {
```

Comment:
this seems like a bad idea, I'd rather wrap the IOException into a MessagingException further down the stack where it's still visible why IOExceptions can happen and why they are handled this way.

Reply:
Good catch. Turns out I removed all code that threw `IOException` but didn't update the method signatures.